python/TestHarness/TestHarness.py

```diff
@@ -207,7 +207,8 @@ def __init__(self, argv, moose_dir, app_name=None):
     Recursively walks the current tree looking for tests to run
     Error codes:
     0x0  - Success
-    0x80 - TestHarness error
+    0x1  - Recoverable errors
+    0x80 - Unrecoverable TestHarness errors
```

Comment:
This comment isn't quite accurate. It should be range based.
0x01 - 0x7F - Recoverable errors
0x80 - 0xFF - Unrecoverable TestHarness errors